### ISSUE TITLE
DialogService: Support multiple kendoDialogContainer

### ISSUE THREAD

**Issue by meriturva** [telerik/kendo-angular#895]
I'm trying to use DialogService and replace our current solution based on bootstrap 4 modal script.
Basically i just try to use DialogService API with different `kendoDialogContainer` and there is a big limitation here. I can only declare just one single kendoDialogContainer so no way to open two different dialogs on two different containers.

See sample: http://plnkr.co/edit/hsw2jRbDyzRLzcmg1mrp?p=preview

What i expected for a service is to have also `kendoDialogContainer` ref passed on [dialogService open method](http://www.telerik.com/kendo-angular-ui/components/dialog/api/DialogService/#toc-open).

**Comment by gyoshev** [comment]
The option is available in the development channel, in release `v1.2.0-dev.201709271213`. It will be included in version `1.2.0` next week -- until then, you can give it a try:

```typescript
    import { Component, ViewChild, ViewContainerRef } from '@angular/core';
 
     import { DialogService } from '@progress/kendo-angular-dialog';
 
     @Component({
       selector: 'my-app',
       template: `
         <button kendoButton (click)="open()">Open dialog</button>
 
         <ng-container #container></ng-container>
       `
     })
     export class AppComponent {
       constructor(private dialogService: DialogService) { }
 
       @ViewChild("container", { read: ViewContainerRef })
       public containerRef: ViewContainerRef;
 
       public open() {
         this.dialogService.open({
           appendTo: this.containerRef,
 
           title: "Please confirm",
           content: "Are you sure?",
           actions: [ { text: "Yes" }, { text: "No" } ]
         });
       }
     }
```

Note that due to Angular specifics, the dialog will be inserted next to the `ViewContainerRef` container.

**Comment by meriturva** [comment]
it is same behaviour i have implemented around bootstrap 4 modal js so will be really easy for me to move to kendo solution. Really cool. I will give it a try next week!

Anyway thanks a lot!
